### PR TITLE
Boiler-Types

### PR DESCRIPTION
**Pull request by giulice**

### CODE REVIEW

**Comment by Ignacio-Sormani** [comment]
Rebase and solve the conflicts before your next push

File: client/.eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
...ingCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"22"},{"filePath":"37","messages":"38","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\index.js",[],["39","40"],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\App.js",[],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\components\\pages\\Companies.jsx",[],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\components\\pages\\Technicians.jsx",[],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\components\\pages\\Buildings.jsx",[],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\components\\pages\\Boilers.jsx",[],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\components\\layout\\sidenav\\Sidenav.jsx",[],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\components\\layout\\header\\Header.jsx",[],"C:\\Radium\\caldar-react\\CaldAR-RR-React\\client\\src\\components\\pages\\Boiler-types\\Boiler-types.jsx",[],{"ruleId":"41","replacedBy":"42"},{"ruleId":"43","replacedBy":"44"},"no-native-reassign",["45"],"no-negated-in-lhs",["46"],"no-global-assign","no-unsafe-negation"]
```
Comment (Ignacio-Sormani) [inline client/.eslintcache]: This file shouldn't be uploaded

File: client/src/App.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+//import Modal from '@material-ui/core/Modal';
 import './App.css';
 import Header from './components/layout/header/Header';
 import Sidenav from './components/layout/sidenav/Sidenav';
 import Buildings from './components/pages/Buildings';
 import Companies from './components/pages/Companies';
 import Boilers from './components/pages/Boilers';
 import Technicians from './components/pages/Technicians';
+import BoilerTypes from './components/pages/Boiler-types/Boiler-types';
```
Comment (Ignacio-Sormani) [inline client/src/App.js]: Make sure that the files and folders names follow the good practices and conventions

File: client/src/components/layout/sidenav/Sidenav.jsx [hunk 3]
```diff
@@ -57,11 +57,22 @@ const Sidenav = () => {
             <NavText style={{ fontSize: '18px' }}>Boilers</NavText>
           </NavItem>
         </Link>
+        <Link
+          to="/boiler-types"
+          style={{ textDecoration: 'none', color: '#f8f8f8' }}
```
Comment (Ignacio-Sormani) [inline client/src/components/layout/sidenav/Sidenav.jsx:62]: Styles should be inside a css file and not inline

Comment (Ignacio-Sormani) [reply client/src/components/layout/sidenav/Sidenav.jsx:62]: Please correct this whenever you see it

File: client/src/components/pages/Boiler-types/Boiler-types.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Boilertypes = () => {
+  return (
+    <div>
+      <h1>Boilers-types Page</h1>
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/Boiler-types/Boiler-types.jsx]: Maybe we should display a title that says: Boiler Types

File: client/src/components/pages/Boiler-types/Boiler-types.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Boilertypes = () => {
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/Boiler-types/Boiler-types.jsx]: Remember to write the name of the components with the first letter of each word with uppercase. For example: BoilerTypes

**Review by tomascravero — APPROVED** [review]

**Review by Alfondalix — APPROVED** [review]

**Review by Ignacio-Sormani — APPROVED** [review]

**Pull request merged**